pom.xml

```diff
@@ -862,6 +896,50 @@
               </execution>
             </executions>
           </plugin>
+           <plugin>
+    <!-- This plugin will set properties values using dependency information -->
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-dependency-plugin</artifactId>
+    <executions>
+      <execution>
+        <goals>
+          <goal>properties</goal>
+        </goals>
+      </execution>
+    </executions>
+  </plugin>
+<plugin>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <version>3.6.1</version>
+    <configuration>
+      <source>1.8</source>
+      <target>1.8</target>
+      <compilerArguments>
+        <Xmaxerrs>10000</Xmaxerrs>
+        <Xmaxwarns>10000</Xmaxwarns>
+      </compilerArguments>
+      <annotationProcessorPaths>
+        <path>
+          <groupId>org.checkerframework</groupId>
+          <artifactId>checker</artifactId>
+          <version>2.8.1</version>
+        </path>
+      </annotationProcessorPaths>
+      <annotationProcessors>
+        <!-- Add all the checkers you want to enable here -->
+        <annotationProcessor>org.checkerframework.checker.index.IndexChecker</annotationProcessor>
+      </annotationProcessors>
+      <compilerArgs>
+        <!-- location of the annotated JDK, which comes from a Maven dependency -->
+        <arg>-Xbootclasspath/p:${annotatedJdk}</arg>
+        <!-- -Awarns turns type-checking warnings into errors. -->
+        <!-- <arg>-Awarns</arg> -->
+        <arg>-AonlyDefs=^org\.apache\.commons\.lang3\.builder\.</arg>
+        <arg>-Astubs=/home/kakarot/commons-lang/Stubs/.</arg>>
```

Comment:
You don't need the `.` at the end of this path.

Moreover, this is an absolute path to a directory on your machine - this cannot possibly work on another computer. Can you make this path relative to the location of this file?